dolphin/db/sqlalchemy/api.py

```diff
@@ -173,7 +173,11 @@ def access_info_create(context, values):
 
 def access_info_update(context, access_info_id, values):
     """Update a storage access information with the values dictionary."""
-    return NotImplemented
+    session = get_session()
+    with session.begin():
+        result = session.query(AccessInfo) \
+            .filter(AccessInfo.storage_id == access_info_id)
+    return result
```

Comment:
it seems that no update operation here

Reply:
Missed while formatting code, Thanks.. Added now..